src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -253,11 +254,19 @@ def post(self, request):
         # Handle interaction actions
         try:
             for action in action_list:
-                if action['name'] == 'status':
+                action_type = action['name']
+
+                analytics.backend.record(
+                    'integrations.slack.action',
+                    action_type=action_type,
+                    actor_id=identity.user.id
```

Comment:
Could this be `identity.user_id` to avoid accidentally querying?

Reply:
dd5e34fe7